.circleci/dockerfiles/rocky-9

```diff
@@ -0,0 +1,11 @@
+FROM rockylinux/rockylinux:9
```

Comment:
need to move this file under the new Github directory

Reply:
ah right